validation-test/compiler_crashers_2_fixed/issue-74545.swift

```diff
@@ -0,0 +1,11 @@
+// RUN: %target-swift-frontend -target %target-swift-5.7-abi-triple -emit-ir %s
```

Comment:
Since this example was crashing in Sema maybe we should just `// RUN: %target-typecheck-verify-swift`?

Reply:
I should probably keep the target, though.